mtp_api/apps/transaction/models.py

```diff
@@ -34,30 +34,48 @@ class Transaction(TimeStampedModel):
 
     # original reference
     reference = models.TextField(blank=True)
-
     received_at = models.DateTimeField(auto_now=False)
 
+    # 6-digit reference code for reconciliation
+    ref_code = models.PositiveIntegerField(null=True)
+
     # set when a transaction is locked and unset if it gets unlocked.
     owner = models.ForeignKey(settings.AUTH_USER_MODEL, null=True, blank=True)
 
     credited = models.BooleanField(default=False)
-
     refunded = models.BooleanField(default=False)
-
     reconciled = models.BooleanField(default=False)
 
     # NB: there are matching boolean fields or properties on the model instance for each
```

Comment:
should we also update the matching model properties for `locked`, `available` and `refund_pending`?